proposals/interface-types/working-notes/record.md

```diff
@@ -0,0 +1,73 @@
+
+
+### Records
+
+Records are collections of named and typed fields. Records can be defined in as
+Interface Types; for example this definition encodes a familiar notion of a
+credit card:
+
+```wat
+(@interface datatype @cc 
+  (record "cc"
+    (ccNo u64)
```

Comment:
The S-Expression convention wasm text format sticks with only uses a fixed set of operators as the first token in an S-Expression.  Also, arbitrary strings (like field names) are usually quoted.  Thus, I think `(ccNo u64)` should be written `(field "ccNo" u64)`.

Reply:
Will adjust accordingly